src/commands/data/schemas/util/syncSchema.ts

```diff
@@ -1,11 +1,11 @@
 /* eslint-disable lines-between-class-members */
 import chalk = require('chalk');
 import * as _ from 'lodash';
+import * as dataServiceRepository from '../../../../repositories/schemas';
```

Comment:
All functions in the dataService class from the file above have been copied to the data service repository. The SDK was added as the first argument every time.

Reply:
Just like the PR for creating functions, could we call this schemaRepository? 🤔